WikiWatershed/model-my-watershed
# MapShed Fix Data Formats and Run GWLF-E

## Overview

Fixes a number of data issues noted in #1319. Finally runs GWLF-E on the actual MapShed data, and returns real results instead of sample values. See commit messages for details.

This depends on the latest version of GWLF-E `0.3.0`. We also switch to pulling from PyPI instead of GitHub.

## Testing Instructions

Reprovision app to pull in latest GWLF-E dependency from PyPI

```bash
$ vagrant provision app
```

Rebundle the scripts:

```bash
$ ./scripts/bundle.sh --debug
```

Debug Celery:

```bash
$ ./scripts/debugcelery.sh
```

Create a project using the Watershed Mult-Year Model, and ensure that there are no errors in Celery, and the output is correctly rendered in both Runoff and Water Quality tabs.

Supersedes #1317 
Connects #1319 
Connects #1321
